dev-aws/kafka-shared/customer-billing/invoice-fulfillment/invoice-fulfillment.tf

```diff
@@ -0,0 +1,21 @@
+resource "kafka_topic" "invoice-fulfillment" {
+  name               = "bex.internal.bill_fulfilled"
```

Comment:
Will we need migration of the data ? Especially for prod?

If we do, we must keep the name of the current topic + a namespace prefix, as we'll use mirror maker 2 to migrate the data.
I guess the namespace prefix is `bex` atm.

Reply:
This is luckily a new topic for a new system, so we don't have to migrate anything.

Our services run in the `customer-billing` namespace, would it be okay if we consistently take the `bex` prefix or should this be a match with the k8s-namespace?

Reply:
I think bex should be fine